TESTS/mbed_drivers/wait_us/main.cpp

```diff
@@ -45,7 +45,7 @@ Case cases[] = {
 };
 
 utest::v1::status_t greentea_test_setup(const size_t number_of_cases) {
-    GREENTEA_SETUP(total_ticks + 5, "timing_drift_auto");
+    GREENTEA_SETUP(total_ticks + 5, "wait_us_auto");
```

Comment:
This is an incorrect change, the host test should be set to `timing_drift_auto`. Please revert this.